consensus/src/main/java/org/apache/iotdb/consensus/ratis/RatisConsensus.java

```diff
@@ -127,6 +133,8 @@ public RatisConsensus(ConsensusConfig config, IStateMachine.Registry registry)
         properties, Collections.singletonList(new File(config.getStorageDir())));
     GrpcConfigKeys.Server.setPort(properties, config.getThisNodeEndPoint().getPort());
 
+    addExecutor = Executors.newSingleThreadExecutor(new IoTThreadFactory("ratis-add"));
```

Comment:
```suggestion
    addExecutor = IoTDBThreadPoolFactory.newCachedThreadPool("ratis-add");
```

Reply:
Should we use cached thread pool here? Or just a single thread pool?

Reply:
done.